plugins/by-name/nvim-ufo/default.nix

```
@@ -130,4 +130,15 @@ lib.nixvim.plugins.mkNeovimPlugin {
         end
       '';
   };
+
+  extraConfig = cfg: {
```

Nit:

```suggestion
  extraConfig = {
```

No need for the `cfg` param if it's unused.

---

Although it will be used if we're about to add a `mkIf`

---

Now `cfg` is used with the added option as suggested